docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+- If the radio link does not connect, follow the [Binding](BINDING.md) procedure.
+- It is HIGHLY recommended to use the same firmware version on both the receiver and Tx module. The latest released firmware is also recommended, especially for new installs.  If your Rx and Tx module firmware versions don't match or are very much out of date, return to the previous step and install the latest mLRS firmware release.
+
+Use the mLRS Lua script to configure your mLRS system
+- Leave your Rx powered on and connected while you complete the following steps.
+- Set the correct RF Band for your region (If using 900 MHz hardware).
+- You can also change the Bind Phrase if desired.
+- Set your desired Mode
+- If you didn't connect sbus or CRSF RC output from your receiver to your flight controller, enter the "Edit Rx" page and set Snd RcChannels to "rc override" (or "rc channels" for ArduPilot 4.6 with RC Channels support).  Return to the main page.  If your radio has a black and white screen, you will need to use the [CLI](CLI.md) for this.
+- Select "Save" and verify that your Rx reconnects.
+- You may wish to review and adjust the Rx and Tx power settings as well as other settings at this time, but the defaults should be OK to get started.
+
+## 6. Optionally, Install the Yaapu Telemetry script
+
+You may wish to install the [Yaapu Telemetry script](https://github.com/yaapu/FrskyTelemetryScript) which uses the CRSF telemetry messages which mLRS generates on your Tx module from the MAVLink data it receives when the Rx Serial Link Mode is set to "MAVLink" or "MAVLinkX".
+
+- If you won't always use a ground station to set MAVLink stream rates when it starts up, be sure to set your ArduPilot [Stream Rate](CRSF.md#stream-rates) parameters.
+
+- See the [Yaapu Wiki](https://github.com/yaapu/FrskyTelemetryScript/wiki) and [mLRS CRSF Telemetry](CRSF.md) page for details.
+
+## 7. Connect a ground station to your mLRS Tx module
+
+There are MAVLink ground station applications available for all common platforms including Windows, MacOS, Linux, Android, and iOS. We won't cover selecting a platform or installing a ground station here.  But we will provide some hints and suggestions for popular connection methods.
+
+Wireless links are possible for all three of the recommended Tx modules.
+
+Bluetooth is very handy for mobile devices and is recommended for the MatekSys Tx modules since they include an HC04 module which is automatically configured by mLRS.  Bluetooth is also supported by the [M5 Stamp Pico Mate](https://shop.m5stack.com/products/m5stamp-pico-diy-kit) which is recommended for use with the [FrSky R9M Tx module](FRSKY_R9.md#esp32-wireless-bridge).
+- The [HC04](MATEKSYS.md#tx-module-hc-04-bluetooth-notes) will be discoverable soon after being powered on.
+- First, pair your HC04 (pin: 1234) or esp32 with your device or computer.  Look for a device which includes "mLRS" in its name.
```

Comment:
... ESP32 ...

can we assume that a new user will know it can be the M5 just mentioned before ?

Reply:
I think I'll move the mention of the M5 down to a bullet item and reword it to include mention of ESP32 and to indicate that not all ESP32 boards support BT.